src/shared/components/data-table-field.tsx

```diff
@@ -310,7 +310,7 @@ export const DataTableField: React.FC<FieldProps> = props => {
 
   return (
     <div className={css.dataTable}>
-      {sensor && <SensorComponent sensor={sensor}/>}
+      {sensor && (sensorFields.length > 0) && <SensorComponent sensor={sensor}/>}
```

Comment:
Good catch, thanks. But I think we should merge this condition `sensorFields.length > 0` into `const sensor` declaration, as this variable is used in a few places (e.g. a few lines below to style table) to check if the sensor is available or not.

Reply:
Makes sense.